invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -86,6 +86,14 @@ def localize_vocabulary_list(field_name, key, attr, obj):
     return localized
 
 
+def record_version(obj):
+    """Return record's version."""
+    field_data = obj.get("metadata", {}).get("version")
+
+    if not field_data:
+        return obj["versions"]["index"]
```

Comment:
On the landing page we have `Version {version}` and that looks fine even when this is used. On the frontend search results if this is used we get: `2010-10-02 (1)` which is maybe a little strange ;) . On Zenodo, it shows up as `v<version>` 
![image](https://user-images.githubusercontent.com/1932651/112048105-1275fe80-8b1c-11eb-89fe-a0541d72fe8b.png)


What about 
```suggestion
        return f"v{obj["versions"]["index"]}"
```
? or something on the frontend...